index.js

```diff
@@ -39,6 +39,10 @@ function warnConfigFile(pkgCwd) {
 	console.log(`${files.join(' & ')} can probably be deleted now that you're using XO.`);
 }
 
+function hasYarn(pkgCwd) {
+	return pathExists.sync(path.join(pkgCwd, 'yarn.lock'));
+}
```

Comment:
@SamVerschueren There's no benefit of it being async as it's not run concurrently with anything. We're not in a server environment, so sync operation can be good, even sometimes better. It will actually be slower with async in this case as the event loop comes with some overhead.